docs/tutorials/basic_commands.md

```diff
@@ -28,32 +31,46 @@ functionalites of the tool. They are:
 * **disable**
 
 In this tutorial, we will go through all those commands to show how to properly use them.
-To achieve that, we will use a Xenial LXD container.
+To achieve that, we will use a Xenial Multipass VM.
 
-## Creating the Xenial LXD container
+## Creating the Xenial Multipass virtual machine
 
-To test all of those commands, let's create a Xenial LXD container. Remember to set up LXD as
-mentioned on the [Prerequisites](#prerequisites) section. After that, just run the command:
+To test all of those commands, we will enable services on the machine. To avoid modifying your
+machine, let's create a VM. Remember to install Multipass as mentioned on the [Prerequisites](#prerequisites)
+section. After that, just run the command:
```

Comment:
```suggestion
The first step in creating our VM is to install Multipass, as mentioned in the
[Prerequisites](#prerequisites) section. After that, just run the command:
```

Reply:
Upon a more detailed reading, the end of the last section and the start of this one are basically the same. I recommend constructing it this way instead (my suggested changes), so that at the end of the last section we are basically saying "this is what we'll do next" and then launch straight into "this is how we do it".